webdriver-spec.html

```diff
@@ -5780,118 +5774,19 @@
  <li><p>Let <var>result</var> be the result of calling <a>execute a function body</a>,
   with arguments <var>body</var> and <var>arguments</var>.
 
- <li><p>Wait for <a>execute a function body</a> to return
-  or for <a>session script timeout</a> milliseconds to expire,
-  whichever happens sooner.
+ <li><p>Let <var>promise</var> be a new Promise.
```

Comment:
speaking to annevk, we shouldnt put anything at the moment as there is a missing dependency in WebIDL. Promises are "assumed knowledge"